multisrc/overrides/madara/randomscan/src/LunaToon.kt

```diff
@@ -7,9 +7,9 @@ import java.text.SimpleDateFormat
 import java.util.Locale
 import java.util.concurrent.TimeUnit
 
-class RandomScan : Madara(
+class LunaToon : Madara(
```

Comment:
You need to match the class name with the `className` you have declared in the generator.

If in `MadaraGenerator.kt` you wrote this:
```kotlin
SingleLang(/* snipped */ className = "LuraToon"),
```

Then the class in overrides must be `LuraToon` and the file name must be `LuraToon.kt`. Please actually understand what you're trying to do and test your changes properly, or any of your future pull requests will be automatically rejected.